test/test.js

```diff
@@ -4,9 +4,7 @@ const domains = require('../email-domain-list.json')
 const extensions = require('../email-extensions.json')
 const assert = require('assert') // core
 const isValidDomain = require('is-valid-domain')
-const checkDomain = require('../check-email.js')
-const wrongDomains = ['.com','.org.uk', '.co.uk']
-const wrongExt = ['gov.uk', ' ', '.co.uk']
+const checkDomain = require('../index.js')
```

Comment:
```suggestion
const checkDomain = require('..')
```

`index.js` is magic (it'll read the `entrypoint` from `package.json`)